Dockerfile

```diff
@@ -40,6 +40,7 @@ RUN apk add --no-cache \
     # needed so non root users can read content of the bundle
     && chmod -R a+r /usr/local/bundle
 
+ENV GO111MODULE=off
```

Comment:
Any reason to specifically change this option?

Reply:
After change this error: 
`go: go.mod file not found in current directory or any parent directory; see 'go help modules'`
appears. This is related to: https://stackoverflow.com/questions/66894200/error-message-go-go-mod-file-not-found-in-current-directory-or-any-parent-dire

Reply:
@nfsec Sorry your saying after you make this change above the error appears or before it was changed that error was appearing? Also this appears to be related to the Go version being bumped, was this part of upgrading Alpine?